src/Compilers/CSharp/Portable/Symbols/FunctionPointers/FunctionPointerParameterSymbol.cs

```diff
@@ -8,7 +8,7 @@
 
 namespace Microsoft.CodeAnalysis.CSharp.Symbols
 {
-    internal sealed class FunctionPointerParameterSymbol : ParameterSymbol
+    internal sealed partial class FunctionPointerParameterSymbol : ParameterSymbol
```

Comment:
>partial [](start = 20, length = 7)

Is this change needed?

Reply:
Not anymore, thought I was going to need to create a specific Cci adapter.

---
In reply to: [374336827](https://github.com/dotnet/roslyn/pull/41340#discussion_r374336827) [](ancestors = 374336827)